tests/features/rhpam/smartrouter/rhpam-smartrouter.feature

```diff
@@ -149,3 +149,16 @@ Feature: RHPAM Smart Router configuration tests
       | mem_limit | 1073741824                                               |
     Then container log should match regex -Xms205m
      And container log should match regex -Xmx819m
+
+  @wip
+  Scenario: Verify if the logging properties is set correctly
+    When container is started with env
+      | variable                | value                   |
+      | SCRIPT_DEBUG            | true                    |
+      | CONFIG_DIR              | /opt/rhpam-smartrouter/ |
```

Comment:
why set this?

Reply:
To configure where is the logging properties to change in the container startup and in test

Reply:
it shouldn't be needed.